MobiusCore/Test/MobiusControllerTests.swift

```diff
@@ -142,6 +142,17 @@ class MobiusControllerTests: QuickSpec {
                             controller.start()
                             expect(controller.running).to(beTrue())
                         }
+                        fit("should ignore events sent while view disposal is pending") {
```

Comment:
This should probably be `it` and not `fit` for master 🙂

Reply:
I was just thinking yesterday that we should lint for this…

Reply:
@JensAyton We lint for it in S4A using [Danger](https://danger.systems) and the following snippet:

```ruby
# Make sure we aren't focusing a specific test and ignoring all others. We don't want to let
# testing shortcuts get into master.
(git.modified_files + git.added_files).each do |file|
  next unless File.file?(file)
  if file.downcase.include?('/tests/') && file.end_with?('.swift')
    contents = File.read(file)
    fail("`xit` or `fit` left in tests (#{file})") if contents =~ /^\w*[xf]it/
    fail("`fdescribe` left in tests (#{file})") if contents =~ /^\w*fdescribe/
    fail("`fitBehavesLike` left in tests (#{file})") if contents =~ /^\w*fitBehavesLike/
  end
end
```